Improve Android bluetoothGatt management (#124)
Navideck/universal_ble: CHANGELOG.md

> ## 0.14.0

Reviewer: ```suggestion
 ## 0.15.0
```

* `getSystemDevices(withServices:)` now sets several generic services by default as filter

> * Fix `getConnectionState` on Android, Now it will return `BleConnectionState.disconnected` if device is connected to System but not with App

Reviewer: ```suggestion
* `getConnectionState` on Android will now return `BleConnectionState.disconnected` if device is connected to the system but not to app
```
